SCsub

```diff
@@ -58,10 +58,10 @@ sources = [
 ]
 
 if env['tools']:
-	with open("tools/godot.d.ts.gen.cpp", "w") as f:
+	with open("tools/godot.d.ts.gen.cpp", "w", encoding="utf-8") as f:
```

Comment:
Python2 does not support this kind of syntax. Which is the default python version on macOS.

Reply:
Godot 3.2 is now only compileable with Python 3.5+
http://docs.godotengine.org/en/3.2/development/compiling/compiling_for_osx.html

Reply:
Not only on Mac OS X, other systems too. So, now there is no need to remake that for Python 2.